hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -156,6 +158,28 @@ def collect_alembic_caches(self, parent_item, project_root):
                 cache_path
             )
 
+    def collect_alembic_caches_geometry(self, parent_item):
+        """
+        Creates items for alembic caches
+
+        :param parent_item: Parent Item instance
+        """
+
+        session_item = parent_item.create_item(
+            "maya.session.geometry",
+            "Alembic Cache",
```

Comment:
Thinking about this more, on the collector side, there's nothing saying this has to be exported as an Alembic cache. We could have multiple publish plugins that attach to this item. One may export an Alembic, another may export an Obj file. Maybe we should call this "Geometry" and give it a new icon? What do you think?